scanomatic/ui_server_data/js/som/fixtures.js

```diff
@@ -373,17 +351,16 @@ function drawHoverSlice(imageCoords) {
 }
 
 export function setCanvas() {
-  selectedFixtureCanvasJq = $(selectedFixtureCanvasId);
-  selectedFixtureCanvasJq.attr('tabindex', '0');
-  [selectedFixtureCanvas] = selectedFixtureCanvasJq;
+  const canvas = $(getSharedValue('selectedFixtureCanvasId'));
```

Comment:
This function required the most rewriting to get it to work with setters/getters. I think I got it right, but please check.